grpc/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>1.0.23</version>
+    <version>1.0.24-SNAPSHOT</version>
```

Comment:
This should help, but #926 takes priority for updating this version on `master`, I guess

Reply:
Yea I will wait until after #926 to merge

Reply:
#926 has been merged